qcodes/dist/schemas/station.schema.json

```diff
@@ -0,0 +1,176 @@
+{
+    "type": "object",
+    "title": "QCoDeS Station configuration validation",
+    "description": "This schema serves the purpose of validating yaml files that describe a QCoDeS station. See more here: https://qcodes.github.io/Qcodes/examples/Station.html#Configuring-the-Station-by-using-a-YAML-configuration-file",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "definitions": {
+        "instruments": {
+            "enum": [
+                ""
+            ]
+        },
+        "parameter_base": {
+            "$comment": "A common base for the properties shared by 'add_parameters' and 'parameters can first be implemented with the new json-schema draft 2019-09, for which there is no implementation at this point. The new feature is called 'unevaluatedProperties'."
```

Comment:
this comment is confusing. what can exactly "can first be implemented with the new json schema"?

Reply:
During my short affair with json-schemas I learned about how their design is a beautiful craftsmanship, just as writing beautiful code. And also here the DRY principle holds true. Currently the schema repeats the same elements for `parameters` and `add_parameters`. This can be refactored with a feature that comes with the new standard that has been released last month, but has not been implemented by the libraries.

Reply:
thanks for the clarification.